cumulusci/cli/tests/utils.py

```diff
@@ -13,6 +15,20 @@ def run_click_command(cmd, *args, **kw):
         return cmd.callback(*args, **kw)
 
 
+def run_cli_command(*args, runtime=None, input=None, **kw):
+    """Run a click command with arg parsing and injected CCI runtime object."""
```

Comment:
Added this as a better way to test click commands, including verifying how the args get parsed and easier ways to provide input and check output. The goal is to gradually switch existing tests over to this and then get rid of run_click_command above.